test/fw/ptl/utils/pbs_logutils.py

```diff
@@ -279,6 +274,31 @@ def convert_date_time(cls, dt=None, fmt=None, syslog=False):
         else:
             return int(tm)
 
+    def get_syslog_date_str(self, logline=None):
```

Comment:
please add a docstring to this function. Same comment for other new functions as well.